### DISCUSSION TITLE
怎么启动截图啊，快捷键多少，设置界面都看不到截图的快捷键，右键也没有！！

### DISCUSSION THREAD

**Discussion by steadyme1** [hiroi-sora/Umi-OCR#583] [General]
怎么启动截图啊，快捷键多少，设置界面都看不到截图的快捷键，右键也没有！！
这个软件安装了怎么用

**Comment by qwedc001** [comment]
你好，在这里。
<img width="802" alt="PixPin_2024-07-09_10-17-26" src="https://github.com/hiroi-sora/Umi-OCR/assets/14366166/b565a18c-1216-4be5-8da4-d7ccd8b548d2">

**Comment by steadyme1** [comment]
![Snipaste_2024-07-11_09-07-06](https://github.com/hiroi-sora/Umi-OCR/assets/175073098/c3b0cb18-9009-43e0-be2e-478ffcd4c451)
没得。。。。。

**Reply by qwedc001** [reply]
你打开的是全局设置，你通过左上的叉号叉掉这个地方，然后点开Umi的截图OCR

**Reply by steadyme1** [reply]
这样啊！！感谢！！
